library/Reporting/Reports/SystemReport.php

```diff
@@ -24,15 +24,24 @@ public function getHtml(Timerange $timerange, array $config = null)
         @$doc->loadHTML($html);
 
         $style = $doc->getElementsByTagName('style')->item(0);
-        $style->parentNode->removeChild($style);
+
+        if ($style) {
+            $style->parentNode->removeChild($style);
+        }
 
         $title = $doc->getElementsByTagName('title')->item(0);
-        $title->parentNode->removeChild($title);
+        if ($title) {
+            $title->parentNode->removeChild($title);
+        }
 
         $meta = $doc->getElementsByTagName('meta')->item(0);
-        $meta->parentNode->removeChild($meta);
+        if ($meta) {
+            $meta->parentNode->removeChild($meta);
+        }
 
-        $doc->getElementsByTagName('div')->item(0)->setAttribute('class', 'system-report');
+        if (($el = $doc->getElementsByTagName('div')->item(0)) !== null) {
```

Comment:
All other blocks also do not set the variable in the `if`, so please do the same here, e.g.

```
$div = ...
if ($div !== null) {
    ...
}